pkg/datastore/ingest.go

```diff
@@ -165,6 +166,7 @@ func (s *Datastore) Ingest(ctx context.Context, datasetID string, name string, c
 
 	if len(docs) == 0 {
 		statusLog.With("status", "skipped").With("reason", "no documents").Debug("No documents loaded")
```

Comment:
Can't we merge both lines then?
I'd say that status==skipped already implies that this file is finished processing, but even if not, I wouldn't make this two log lines then